platformio.ini

```diff
@@ -52,11 +52,8 @@ upload_speed = 921600
 
 ; Uncomment below if you want to build for esp32
 ; Check your board name at https://docs.platformio.org/en/latest/platforms/espressif32.html#boards
-; [env:esp32]
-; lib_deps =
-;   ${env.lib_deps}
-;   lorol/LittleFS_esp32 @ 1.0.6
-; platform = espressif32 @ 3.5.0
-; board = esp32dev
+;[env:esp32]
+;platform = espressif32
```

Comment:
Please set the minimum required version via `@ ^1.23.4` or the exact version via `@ 1.23.4`